htconfig.php

```diff
@@ -91,3 +91,6 @@
 
 // Allowed protocols in link URLs; HTTP protocols always are accepted
 $a->config['system']['allowed_link_protocols'] = array('ftp', 'ftps', 'mailto', 'cid', 'gopher');
+
+// Authentication cookie lifetime, in days
+$a->config['system']['auth_cookie_lifetime'] = 7
```

Comment:
Please add the variable to the [documentation](https://github.com/friendica/friendica/blob/develop/doc/htconfig.md) of things that can be set in the `.htconfig.php` file. And please mention the default value.

Also if you want to have this included in the config file after the installation you need to include it into the template file (/view/templates/htconfig.tpl)